tools/scope_download_classification.py

```diff
@@ -174,4 +174,4 @@ def download_classification(file: str, gloria, group_ids: list, token: str):
     args = parser.parse_args()
 
     # download object classifications in the file
-    download_classification(args.file, gloria, args.group_ids, args.token)
+    download_classification(args.file, gloria, args.group_ids, args.token)
```

Comment:
Should there be a newline at the end of this file?

Reply:
Yes. I had overlooked the "no newline" error earlier.